rpc/wrtc_signaling_answerer.go

```diff
@@ -303,37 +306,50 @@ func (ans *webrtcSignalingAnswerer) answer(client webrtcpb.SignalingService_Answ
 			return err
 		}
 
-		var callFlowWG sync.WaitGroup
+		var pendingCandidates sync.WaitGroup
 		waitOneHost := make(chan struct{})
 		var waitOneHostOnce sync.Once
-		pc.OnICECandidate(func(i *webrtc.ICECandidate) {
+		pc.OnICECandidate(func(icecandidate *webrtc.ICECandidate) {
```

Comment:
Also I don't think I mentioned this before, but I really appreciate you renaming the ice candidate variable away from `i`! I have a 4-5 line limit where I'm ok with single letter variables.

Reply:
hah, also my brain is hardwired to treat `i` as an index so it had to change for my sanity. i think i'll do a flyby change for this in the client code as well.